.travis.yml

```
@@ -10,35 +10,51 @@ node_js:
 
 cache: npm
 
+env:
+  jobs:
+    - BUILD_SOUP=true
+
 before_install:
   - if [[ `npm -v` != 6* ]]; then npm install -g npm@6; fi
 
 install:
   - npm install
+  - npm install --save-dev ci-toolbox@2.1.1
 
 deploy:
   # Control deployment by setting a value for `on`. Setting the `branch`
   # option to `master` means Travis will only attempt a deployment on
   # builds of your repo's master branch (e.g., after you merge a PR).
-  on:
-    tags: true
-    node_js: 10.15.3
-#    branch: dbl
-  provider: s3
-  # You can refer to environment variables from Travis repo settings!
-  access_key_id: $AWS_ACCESS_KEY_ID
-  secret_access_key: $AWS_SECRET_ACCESS_KEY
-  region: $AWS_DEFAULT_REGION
-  # Name of the S3 bucket to which your site should be uploaded.
-  bucket: $AWS_BUCKET
-  # Prevent Travis from deleting your built site so it can be uploaded.
-  skip_cleanup: true
-  # Path of the source directory containing your built site.
-  local_dir: deploy
-  # Path to a directory containing your built site.
-  upload-dir: deploy
-  # Set the Cache-Control header.
-  cache_control: "max-age=21600"
+  - provider: s3
+    on:
+      tags: true
+      node_js: 10.15.3
+    # You can refer to environment variables from Travis repo settings!
+    access_key_id: $AWS_ACCESS_KEY_ID
+    secret_access_key: $AWS_SECRET_ACCESS_KEY
+    region: $AWS_DEFAULT_REGION
+    # Name of the S3 bucket to which your site should be uploaded.
+    bucket: $AWS_BUCKET
+    # Prevent Travis from deleting your built site so it can be uploaded.
+    skip_cleanup: true
+    # Path of the source directory containing your built site.
+    local_dir: deploy
+    # Path to a directory containing your built site.
+    upload-dir: deploy
+    # Set the Cache-Control header.
+    cache_control: "max-age=21600"
+  # Deploy SOUP list
+  - provider: s3
+    on:
+      tags: true
```

Do we need to add the condition on the node version? otherwise we may publish it for each node version that is running on travis
```suggestion
      tags: true
      node_js: 10.15.3
```

---

yes good catch as I missed that :)